zip-0221.rst

```diff
@@ -711,8 +711,12 @@ probability that a dishonest chain commitment will be accepted by the verifier.
 security analysis assumes adversary mining power is bounded by a known fraction of
 combined mining power of honest nodes, and cannot drop or tamper with messages between
 client and full nodes. It also assumes the client is connected to at least one full node
-and knows the genesis block. However, these security properties have not been examined
-closely in chain models with rapidly adjusting difficulty.
+and knows the genesis block.
+
+In addition, [#FlyClient]_ only analyses these security properties in chain models with
+slowly adjusting difficulty, such as Bitcoin. It leaves their analysis in chains with
+rapidly adjusting difficulty –such as Zcash or Ethereum– as an open problem, and states
+that the FlyClient protocol provides only heuristic security guarantees in that case.
```

Comment:
The Rationale section states:
> These commitments enable FlyClient in the variable-difficulty model. Specifically, they
allow light clients to reason about application of the difficulty adjustment algorithm
over a range of blocks. They were chosen via discussion with an author of the FlyClient
paper.

Make reference to that here, so this paragraph covers the following points:
- The paper leaves this as an open problem.
- This ZIP makes an informed choice as to how to adapt for this.
- The choice, while informed by an author of the paper, has not been analysed in academia.
- If subsequent analysis were to find that FlyClient + the ZIP 221 adaptions were insufficient, a future NU could modify or replace ZIP 221.